plugin/plugin_remove_all_consoleLogs.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function ({ types: t }) {
+	return {
+		visitor: {
+			CallExpression(path) {
+				const expectedPattern = "console";
+				const expectedProp = "log";
+				const callee = path.get("callee");
+
+				if (callee.matchesPattern(expectedPattern, true)) {
```

Comment:
Maybe should `expectedPattern` be regEx?
Check, please.

Reply:
done